src/site/_data/i18n/tags.yml

```diff
@@ -298,6 +298,24 @@ cors:
     ru: Последние новости и статьи о CORS.
     zh: 我们关于 CORS 的最新资讯、动态和故事。
 
+cross-browser:
+  title:
+    en: Cross-browser
+    es: Multi-navegador
+    ja: クロスブラウザ
+    ko: 브라우저 간
+    pt: Vários navegadores
+    ru: кроссбраузерный
+    zh: 跨浏览器
+  description:
+    en: Our latest news, updates, and stories about features that are supported cross-browser.
```

Comment:
```suggestion
    en: Our latest news, updates, and stories about features that have become supported cross-browser.
```

Reply:
Ideally everything on this site is supported cross browser, the aim of this tag is to highlight the recent things (I'll likely remove it as things get to be a year old or something).